pilot/pkg/networking/core/v1alpha3/cluster_builder_test.go

```diff
@@ -1217,7 +1232,7 @@ func TestBuildLocalityLbEndpoints(t *testing.T) {
 					Service:     service,
 					ServicePort: servicePort,
 					Endpoint: &model.IstioEndpoint{
-						Address:      "192.168.1.2",
+						Addresses:    []string{"192.168.1.2"},
```

Comment:
TestBuildLocalityLbEndpoints needs dual-stack tests

Reply:
Already add additional addresses unit tests for `TestBuildLocalityLbEndpoints` and `TestConcurrentBuildLocalityLbEndpoints`